frontend/uddhar-frontend/src/public/components/Footer.jsx

```
@@ -0,0 +1,411 @@
+                    Email{" "}
+                  </label>
+
+                  <div className="p-2 sm:flex sm:items-center sm:gap-4">
+                    <input
+                      type="email"
+                      id="UserEmail"
+                      placeholder="john@rhcp.com"
+                      className="w-full h-12 pl-3 border-none focus:border-transparent focus:ring-transparent sm:text-sm dark:bg-gray-900 dark:text-white"
+                    />
+
+                    <button className="mt-1 w-full bg-amber-500 cursor-pointer px-6 py-3 text-sm font-bold tracking-wide text-white uppercase transition-none hover:bg-amber-600 sm:mt-0 sm:w-auto sm:shrink-0">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)